src/script/view_model/AuthViewModel.js

```diff
@@ -1424,15 +1432,11 @@ class AuthViewModel {
    * Validate password input.
    *
    * @private
-   * @param {z.auth.AuthView.MODE} mode - View state of the authentication page
+   * @param {string} password - Password to validate
    * @returns {undefined} No return value
    */
-  _validate_password(mode) {
-    if (this.password().length < z.config.MINIMUM_PASSWORD_LENGTH) {
-      const isPasswordVerification = mode === z.auth.AuthView.MODE.VERIFY_PASSWORD;
-      const errorMessage = isPasswordVerification ? t('authErrorPasswordWrong') : t('authErrorPasswordShort');
-      this._add_error(errorMessage, z.auth.AuthView.TYPE.PASSWORD);
-    }
+  _validatePassword(password) {
+    return new RegExp(ValidationUtil.getNewPasswordPattern()).test(password);
```

Comment:
Let's use the min password length config from the server here

Reply:
Yep. When you merge https://github.com/wireapp/wire-webapp/pull/6346#pullrequestreview-225547391 I'll rebase and consume it